utilities/rocAL/rocAL_unittests/testAllScripts.sh

```diff
@@ -85,78 +85,78 @@ do
         ./rocAL_unittests 0 "$image_path" "${output_path}Flip_${rgb_name[$rgb]}_${device_name}" $width $height 47 $device $rgb 0 $display
 
         # coco detection
-        ./rocAL_unittests 2 "$coco_detection_path" "${output_path}Gamma_${rgb_name[$rgb]}_${device_name}" $width $height 33 $device $rgb 0 $display
-        ./rocAL_unittests 2 "$coco_detection_path" "${output_path}Contrast_${rgb_name[$rgb]}_${device_name}" $width $height 34 $device $rgb 0 $display
-        ./rocAL_unittests 2 "$coco_detection_path" "${output_path}Vignette_${rgb_name[$rgb]}_${device_name}" $width $height 38 $device $rgb 0 $display
+        ./rocAL_unittests 0 "$image_path" "${output_path}Gamma_${rgb_name[$rgb]}_${device_name}" $width $height 33 $device $rgb 0 $display
+        ./rocAL_unittests 0 "$image_path" "${output_path}Contrast_${rgb_name[$rgb]}_${device_name}" $width $height 34 $device $rgb 0 $display
+        ./rocAL_unittests 0 "$image_path" "${output_path}Vignette_${rgb_name[$rgb]}_${device_name}" $width $height 38 $device $rgb 0 $display
```

Comment:
@fiona-gladwin Why are we using same case 0 for all test cases? In that case the comments are all misleading

Reply:
We have not given support for all readers in this PR yet. Since we want to check the functionalities of all augmentations currently this unit test will run and compare against Golden outputs using File source reader. 

In the readers PR we will revert back to our TOT unit test which tests all readers + augmentations.